{
  "comments": [
    {
      "key": {
        "uuid": "ee093ece_2f8e6251",
        "filename": "src/laas_dashboard/settings.py",
        "patchSetId": 3
      },
      "lineNbr": 63,
      "author": {
        "id": 7081
      },
      "writtenOn": "2021-09-13T14:46:47Z",
      "side": 1,
      "message": "re-indent comment",
      "revId": "e434bee5b33a09807575c691fbec781356690bdd",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "9c1eeec3_29c29002",
        "filename": "src/laas_dashboard/settings.py",
        "patchSetId": 3
      },
      "lineNbr": 80,
      "author": {
        "id": 7081
      },
      "writtenOn": "2021-09-13T14:46:47Z",
      "side": 1,
      "message": "this should just be an \"else\" saying \"auth_setting set to an unsupported option: {option}\"",
      "revId": "e434bee5b33a09807575c691fbec781356690bdd",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": true
    }
  ]
}